Gemfile

```diff
@@ -81,6 +80,10 @@ gem "sync",                           "~>0.5",         :require => false
 gem "sys-filesystem",                 "~>1.3.4"
 gem "terminal",                                        :require => false
 
+# Custom gem that replaces mime-types in order to redirect mime-types calls to mini_mime
+#   Source is located at https://github.com/ManageIQ/mime-types-redirector
+gem "mime-types",                     "~>3.0",     :source => "https://rubygems.manageiq.org", :require => false
```

Comment:
Why add the `:require => false`?

Reply:
Oh, huh...I thought original _was_ `require => false`.

Should I remove it?  In the end we probably want require false anyway.

Reply:
I'm fine with it for now.  We can change it later if it's an issue. (especially since it will delay it further waiting for Travis if we change it)